tests/conftest.py

```diff
@@ -248,29 +248,31 @@ def application_docker_image_configs() -> Mapping[str, Dict[str, Any]]:
             "": dict(
                 buildargs={
                     "NODE_RUNTIME_FLAGS": "--perf-prof --interpreted-frames-native-stack",
+                    "NODE_IMAGE": "node",
                     "NODE_IMAGE_TAG": "node@sha256:59531d2835edd5161c8f9512f9e095b1836f7a1fcb0ab73e005ec46047384911",
```

Comment:
This means we have `node:node@sha256:59531d2835edd5161c8f9512f9e095b1836f7a1fcb0ab73e005ec46047384911` as the image, is that the way it works? Weird o.O

Reply:
Oh, didn't notice that. I will change it